api/stream/design.go

```diff
@@ -344,6 +344,19 @@ var _ = Service("stream", func() {
 		})
 	})
 
+	Method("SourceVersionDetail", func() {
+		Meta("openapi:summary", "Source version detail")
+		Description("Source version detail.")
+		Result(EntityVersion)
+		Payload(SourceVersionDetailRequest)
+		HTTP(func() {
+			GET("/branches/{branchId}/sources/{sourceId}/versions/{version_number}")
```

Comment:
`{versionNumber}` for consistency.

Reply:
[01bde4f](https://github.com/keboola/keboola-as-code/pull/2087/commits/01bde4f2e4982529ebc73bf07555954d24a9cdfa)